data_source_obmcs_core_instances_test.go

```diff
@@ -52,9 +52,7 @@ func (s *DatasourceCoreInstanceTestSuite) SetupTest() {
 
 	data "oci_core_images" "t" {
 		compartment_id = "${var.compartment_id}"
-		operating_system = "Oracle Linux"
-		operating_system_version = "7.4"
-		limit = 1
+		display_name = "Oracle-Linux-7.4-2017.10.25-0"
```

Comment:
can something like `display_name = "${var.InstanceImageDisplayName}"` not be used here (like you've done in the other cases)

Reply:
I was just following the existing pattern in each test/example, where some use a variable and some hardcode.